integration_test/cases/browser/set_value_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Wallaby.Integration.Browser.SetValueTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "forms.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "set_value/3" do
+    test "accepts a query", %{page: page} do
+      page
+      |> send_keys(Query.text_field("Name"), ["Chris", :tab, "c@keathley.io"])
+
+      assert page
+      |> find(Query.text_field("Name"))
+      |> has_value?("Chris")
```

Comment:
sorry we don't have a code linter setup yet, but I think indentation wise we'd like this rather to be:

```
assert page
       |> call
       |> call
```

(goes for all stuff here) :)